monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -3,9 +3,10 @@ import {Route} from 'react-router-dom';
 import {Col, Nav} from 'react-bootstrap';
 import AuthComponent from '../AuthComponent';
 import MustRunMonkeyWarning from '../report-components/common/MustRunMonkeyWarning';
-import AttackReport from '../report-components/AttackReport'
-import SecurityReport from '../report-components/SecurityReport'
-import ZeroTrustReport from '../report-components/ZeroTrustReport'
+import AttackReport from '../report-components/AttackReport';
+import RansomwareReport from '../report-components/RansomwareReport';
+import SecurityReport from '../report-components/SecurityReport';
+import ZeroTrustReport from '../report-components/ZeroTrustReport';
```

Comment:
Maybe not important at all but can we just order the imports like the section order?